packages/wonder-blocks-form/src/components/__docs__/labeled-text-field.argtypes.js

```diff
@@ -0,0 +1,248 @@
+// @flow
+
+export default {
+    id: {
+        description: `An optional unique identifier for the TextField.
```

Comment:
One thing that I'm not completely sure: you might not need to add descriptions here. I took a look at the labeled-text-field prop types, and it looks like Storybook might be able to get the description directly from the JSDocs, mostly because these definitions are located within the same file (again, not completely sure about this).

Reply:
Hmm... it seems kind of inconsistent? It pulls them fine sometimes but it wasn't able to here. I think it wasn't working in this case because `TextField` is a different element than `TextFieldInternal`, and the props are under `TextFieldInternal`. I had to add the Usage comment above the `TextField` at the bottom of the file too.